### PR TITLE
Rework 3

### PR DESCRIPTION
**Pull request by ypid**
Related to: https://github.com/debops-contrib/ansible-apache

@drybjed If you want you can move [the apache role](https://github.com/debops-contrib/ansible-apache) to DebOps core.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -330,11 +333,18 @@ nginx_ocsp_resolvers: []
 
 # HTTP Strict-Transport-Security
 # https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security
+# Maximum age in seconds for which clients should remember to only make secure
+# connections.
+# Defaults to six earth month.
```
Comment (drybjed) [inline defaults/main.yml]: Getting ready for a Mars colony, I see... Good idea, we need to make sure that DebOps can work on Mars. Go SpaceX!

Comment (ypid) [inline defaults/main.yml]: I just copied https://bettercrypto.org/ :wink: Should I remove it?

File: templates/etc/ansible/facts.d/nginx.fact.j2 [hunk 2]
```diff
@@ -29,6 +29,7 @@ cat << EOF
 {% if nginx_tpl_default_server_ssl %}
 "default_server_ssl": "{{ nginx_tpl_default_server_ssl }}",
 {% endif %}
+"enabled": {{ (nginx__deploy_state in [ 'present' ]) | lower }},
```
Comment (drybjed) [inline templates/etc/ansible/facts.d/nginx.fact.j2]: Shouldn't this use a `ternary()` filter to have a boolean result?

File: defaults/main.yml (hunk 1 above)
Comment (drybjed) [inline defaults/main.yml]: No, it's fine, although I would write it as "Defaults to six Earth months." I guess it's an issue with source material.

File: templates/etc/ansible/facts.d/nginx.fact.j2 (hunk 2 above)
Comment (ypid) [inline templates/etc/ansible/facts.d/nginx.fact.j2]: Why? `string in list` is already a comparison which results in an boolean.

**Pull request merged**